src/object/srv_obj_migrate.c

```diff
@@ -650,19 +703,44 @@ migrate_dkey(struct migrate_pool_tls *tls, struct migrate_one *mrone)
 		tls->mpt_pool_hdl = ph;
 	}
 
-	/* Open client dc handle */
+	/* Open the destination *server* container locally, which will create
+	 * it if it does not exist
+	 */
```

Comment:
we only needs to do this for mpt_clear_conts right? These are pure overhead for rebuild.

Reply:
The new version of this is much cleaner - I'd like to know if you still think we need to find a way not to do it for rebuild